src/ontology/pato-edit.obo

```diff
@@ -21836,6 +21836,14 @@ def: "A degree of pigmentation quality that is diversified with patches, or irre
 is_a: PATO:0002248 ! pigmented
 created_by: https://orcid.org/0000-0002-3528-5267
 
+[Term]
+id: PATO:0070001
+name: salinity
+def: "An osmolarity quality inhering in a bearer by virtue of the total amount of ionic salts dissolved in the bearer." [http://db0nus869y26v.cloudfront.net/en/Salinity, https://en.wikipedia.org/wiki/Salinity, https://rwu.pressbooks.pub/webboceanography/chapter/5-3-salinity-patterns/, https://www.sciencedirect.com/referencework/9780080983004/treatise-on-geochemistry, https://www.sciencedirect.com/science/article/pii/B9780857096807500012]
```

Comment:
If you are referring a book, I think the preferred format is to xref the ISBN in the format ISBN:XXXXXXXXXXX

Reply:
otherwise this looks good to me

Reply:
Agree with Nicole. Switch the one book to ISBN and then this is good to go.